docs/TESTS.md

```diff
@@ -14,16 +14,33 @@ It will need the following:
 1. At least one valid SemVer tag in the format `x.y.z`
 [terraform-random-module](ttps://github.com/caseylang/terraform-random-module) is a good example repo. 
    
-### 2. Set up environment variables
+## 2. Set up environment variables (ENVVARS)
```

Comment:
Should we just suggest using the script being added here? https://github.com/hashicorp/go-tfe/pull/399

Reply:
Sure! I'll open a new PR with that update.